src/screens/Messages/Conversation/ChatDisabled.tsx

```diff
@@ -20,7 +30,136 @@ export function ChatDisabled() {
             access to chats on Bluesky.
           </Trans>
         </Text>
+        <AppealDialog />
       </View>
     </View>
   )
 }
+
+function AppealDialog() {
+  const control = Dialog.useDialogControl()
+  const t = useTheme()
+
+  const {
+    state: pressed,
+    onIn: onPressIn,
+    onOut: onPressOut,
+  } = useInteractionState()
+
+  return (
+    <>
+      <Text
+        style={[
+          a.mt_sm,
+          a.text_sm,
+          a.leading_snug,
+          {color: t.palette.primary_500},
+          pressed && {
+            ...web({outline: 0}),
+            textDecorationLine: 'underline',
+            textDecorationColor: t.palette.primary_500,
+          },
+        ]}
+        onPress={control.open}
+        onPressIn={onPressIn}
+        onPressOut={onPressOut}>
+        <Trans>Appeal this decision</Trans>
+      </Text>
```

Comment:
Rn this isn't a tabbable element, so not good for a11y. This is the hacky way we have atm to add an inline link style button: https://github.com/bluesky-social/social-app/blob/main/src/components/dms/MessageItem.tsx#L222

Reply:
Basically it's a link with an `onPress` that disables the link handling

Reply:
Could alternatively just use a `small` sized `Button`

Reply:
I think a small button is the way to go here